GSA-TTS/identity-site
# Add initializer script to base layout

## 🛠 Summary of changes

Includes the design system initializer script to the base layout.

Notably, this can reduce flickering appearance of the banner while the page is loading.

See also: https://designsystem.digital.gov/components/banner/#using-the-banner-component-2

This is newly available as of `@18f/identity-design-system@9.0.0` (see [release notes](https://github.com/18F/identity-design-system/releases/tag/v9.0.0)), so the branch currently merges to `aduth-rm-prose-styles`, where that upgrade is happening.

## 📜 Testing Plan

1. Go to https://federalist-17bd62cc-77b7-4687-9c62-39b462ce6fd5.sites.pages.cloud.gov/preview/gsa-tts/identity-site/aduth-init-script/
2. Reload a few times with caching disabled (either hard refresh or refresh while Chrome DevTools Network tab is open and "Disable cache" is checked)
3. Observe that there's no noticeable flickering of the banner while the page is loaded
